css/editCompetition.import.less

```diff
@@ -0,0 +1,24 @@
+/*
+ * Google maps input is tiny! Use styling from:
+ * https://developers.google.com/maps/documentation/javascript/examples/places-searchbox
+ */
+#competitionLocationMapInput {
+  background-color: #fff;
+  padding: 2px 11px 2px 13px;
+  width: 400px;
+  font-family: Roboto;
+  font-size: 15px;
+  font-weight: 300;
+  text-overflow: ellipsis;
+  margin-top: 20px;
+}
+#competitionLocationMapInput:focus {
```

Comment:
Yay, I can finally make you learn parent selectors! Nest this bad boy and use a parent selector instead :)

http://lesscss.org/features/#parent-selectors-feature